src/doc/gui_manual/Operators/OperatorTypes/PICS/Poincare_operator.rst

```diff
@@ -0,0 +1,318 @@
+coordinates). The basis of constructing a connected plot is to accurately
+determine the number of toroidal and poloidal windings (i.e. the winding pair).
+This process is iterative, starting with a minimum number of puncture points
+and continuing until the toroidal and poloidal windings are known or the
+maximum number of punctures is reached. If an accurate winding pair is
+determined, then the puncture points are connected based on it. For more
+information, refer to the following resources.
+
+A.R. Sanderson, G. Chen, X. Tricoche, E. Cohen. “Understanding Quasi-Periodic
+Fieldlines and Their Topology in Toroidal Magnetic Fields,” In Topological
+Methods in Data Analysis and Visualization II, Edited by R. Peikert and H.
+Carr and H. Hauser and R. Fuchs, Springer, pp. 125--140. 2012.
+
+A.R. Sanderson, G. Chen, X. Tricoche, D. Pugmire, S. Kruger, J. Breslau.
+“Analysis of Recurrent Patterns in Toroidal Magnetic Fields,” In Proceedings
+Visualization / Information Visualization 2010, IEEE Transactions on
+Visualization and Computer Graphics, Vol. 16, No. 4, pp. 1431-1440. 2010.
+
+
+Source
+^^^^^^
+
+Source Type
+"""""""""""
+
+The user can seed the fieldlines either as a single point or a series of points
+along a line.
+
+Point
+    Seed from a point. The point location can be set interactively using the
+    interactive point tool. Parameters are: 
+
+    * Location - Cartesian point with three values separated with a space. 
+
+Point List
+    Seed from a list of points.
+
+    * Add Point - add a point which can then be edited by the user. Double
+      clicking on the entry enters the editing mode.
+    * Delete Point - delete the currently selected entry.
+    * Delete All Points - delete all the points in the list.
+    * Read Text File - read in a list of points from a text file. The format
+      must have one point per line as either "X Y Z" or "X, Y, Z"
+
+Line
+    Seed from a line defined by its end points. The line location can be set
+    interactively using the interactive line tool.
+ 
+    * Start - X Y Z location of the line starting point.
+    * End - X Y Z location of the line ending point.
+    * Sampling - Specify the type of sampling and the number of samples.
+
+Note: if the Field is set to be the M3D-C1 integrator the point locations will
+be converted from Cartesian to Cylindrical coordinates. And in the 2D case, phi
+will be set to 0. 
+
+Field
+"""""
+
+See Field attributes that are common to all PICS operators.
+
+
+Integration
+^^^^^^^^^^^
+
+Specify settings for numerical integrators. In addition to the Integration
+attributes common to all PICS operators, the Poincare operator supports the
+following attributes.
+
+Punctures
+"""""""""
+
+While integrating the streamline to be used the for Poincaré plot, the user has
+the option to require a minimum number of initial puncture points for the
+analysis. Similarly they may limit it in case of run-a-way fieldlines that can
+not be fully analyzed.
```

Comment:
This is confusing.

Reply:
Definitely. I'm still working on a way to write this better. A Poincare curve is a closed loop. Numerically, the close is never perfect, so we have approximate methods to determine if the loop is closed, and the puncture points options is one of those methods.